source/includes/oai-pmh/_root.md

```diff
@@ -124,14 +157,25 @@ We support both harvesting of the _entire repository_ as well as _selective
 harvesting_ of communities.
 
 
-**Entire repository**
-
-In order to harvest the entire repository you make an OAI-PMH request without
-passing any set specification.
+### Entire repository
 
 [See example](https://zenodo.org/oai2d?verb=ListRecords&
 metadataPrefix=oai_datacite)
```

Comment:
We should also have a banner informing them and pointing to the rate limit.
Could be addressed in a separate PR.